port/common/omrport.tdf

```diff
@@ -673,6 +673,269 @@ TraceExit=Trc_PRT_sl_open_shared_library_Exit3 Group=sl Overhead=1 Level=10 NoEn
 
 TraceEvent=Trc_PRT_sl_open_shared_library_Event2 Group=sl Overhead=1 Level=10 NoEnv Template="omrsl_open_shared_library OS error message: %s"
 
+// omrshmem trace points
+TraceEntry=Trc_PRT_shmem_omrshmem_createFile_Entry Obsolete Group=omrshmem Overhead=1 Level=1 NoEnv Template="omrshmem createFile: Entry"
+TraceExit=Trc_PRT_shmem_omrshmem_createFile_Exit1 Obsolete Group=omrshmem Overhead=1 Level=1 NoEnv Template="omrshmem createFile: base file already exists!"
+TraceExit=Trc_PRT_shmem_omrshmem_createFile_Exit2 Obsolete Group=omrshmem Overhead=1 Level=1 NoEnv Template="omrshmem createFile: base file creation error: errno = %s"
+TraceExit=Trc_PRT_shmem_omrshmem_createFile_Exit Obsolete Group=omrshmem Overhead=1 Level=1 NoEnv Template="omrshmem createFile: Normal Exit"
```

Comment:
New tracepoints should added at the end of the file: fix #5523.

Reply:
Noted. Thank You @keithc-ca .